src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -239,6 +241,19 @@ export class BundleDataClient {
       );
 
     const data = persistedData[0].data;
+
+    // Post-populate any missing message hashes.
+    // @todo This can be removed once the legacy types hurdle is cleared (earliest 7 days post migration).
+    Object.values(data.bundleDepositsV3).forEach((x) =>
```

Comment:
Can we unit test this?

Reply:
I've added a test for it in the subsequent PR here: https://github.com/across-protocol/sdk/pull/850/commits/1a6311b2fb6a34e72bdfef4beb924fa2c8adc5b3